big-practice/src/assets/css/update-customer.css

```diff
@@ -0,0 +1,174 @@
+.modal-update {
+  width: 968px;
+  height: 797px;
+  position: absolute;
+  top: 290%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1px solid black;
+  border-radius: 20px;
+  padding-top: 52px;
+  padding-left: 50px;
+  padding-right: 47px;
+}
+
+.modal-header {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+}
+
+.title {
+  color: var(--primary-color);
+  font-size: 36px;
+  font-family: var(--primary-font);
+  font-style: normal;
+  font-weight: var(--fw-semibold);
+  line-height: normal;
+  letter-spacing: 0.36px;
+  padding-bottom: 52px;
+}
+
+.sign-form {
+  display: grid;
+  grid-template-columns: auto auto;
+}
+
+.name-customer {
+  color: var(--accent-color);
+  font-size: 20px;
+  font-family: var(--secondary-font);
+  font-style: normal;
+  font-weight: var(--fw-bold);
+  line-height: 24px;
+  padding-bottom: 19px;
+}
+
+.fname {
+  width: 413px;
+  height: 62px;
+  border: 1px solid black;
+  border-radius: 5px;
+  margin-bottom: 42px;
+  padding-left: 10px;
+  font-size: 16px;
+}
+
+.status-form {
+  padding-right: 34px;
+  color: var(--secondary-color);
+  font-size: 20px;
+  font-family: var(--secondary-font);
+  font-style: normal;
+  font-weight: var(--fw-medium);
+  line-height: 24px;
+}
+
+.btn {
+  width: 264px;
+  height: 62px;
+  color: #FFF;
+  font-size: 20px;
+  font-family: var(--secondary-font);
+  font-style: normal;
+  font-weight: var(--fw-semibold);
+  line-height: 24px;
+  border: none;
+  border-radius: 5px;
+}
+
+.btn-modal-update {
+  display: flex;
+  justify-content: center;
+  padding-top: 115px;
+}
+
+.submit {
+  background-color: #377def;
+  margin-right: 64px;
+}
+
+.cancel {
+  background-color: var(--secondary-color);
+}
+
+/* modal delete */
+.modal-del {
+  width: 968px;
+  height: 337px;
+  position: absolute;
+  top: 590%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1px solid black;
+  border-radius: 20px;
+  padding-top: 52px;
+  padding-left: 50px;
+  padding-right: 47px;
+}
+
+.btn-del {
+  padding-top: 63px;
+}
+
+.status {
+  display: flex;
+  align-items: center;
+}
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 56px;
+  height: 26px;
+}
+
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 22px;
+  width: 22px;
+  left: 3px;
+  bottom: 3px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .slider {
+  background-color: #2196F3;
+}
+
+input:checked + .slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(26px);
```

Comment:
```
-webkit-transform: translateX(26px);
 -ms-transform: translateX(26px);
 transform: translateX(26px);
 ```
Have you ever used any CSS Autoprefixer tools?

Reply:
I didn't ever use CSS Autoprefixer tools. I will ping you to discuss these tools.

Reply:
Yes, we will have a call. You can refer to these docs:
- https://autoprefixer.github.io/
- https://parceljs.org/languages/css/#postcss